src/AAVE/Card/AssetsToSupply.jsx

```diff
@@ -0,0 +1,212 @@
+                          <img
+                            width={64}
+                            height={64}
+                            src={`https://app.aave.com/icons/tokens/${row.symbol.toLowerCase()}.svg`}
+                          />,
+                          <div>
+                            <div className="token-title">{row.symbol}</div>
+                            <div className="token-chain">{row.name}</div>
+                          </div>,
+                        ],
+                      }}
+                    />,
+                    <div>
+                      <div>{row.balance}</div>
+                      <div>$ {row.balanceInUSD}</div>
+                    </div>,
+                    `${(Number(row.supplyAPY) * 100).toFixed(2)} %`,
+                    <div style={{ paddingLeft: "50px" }}>
+                      {row.usageAsCollateralEnabled && (
+                        <img
+                          src={`${config.ipfsPrefix}/bafkreibsy5fzn67veowyalveo6t34rnqvktmok2zutdsp4f5slem3grc3i`}
+                          width={16}
+                          height={16}
+                        />
+                      )}
+                      {!row.usageAsCollateralEnabled && (
+                        <img
+                          src={`${config.ipfsPrefix}/bafkreie5skej6q2tik3qa3yldkep4r465poq33ay55uzp2p6hty2ifhkmq`}
+                          width={16}
+                          height={16}
+                        />
+                      )}
+                    </div>,
+                    <Widget
+                      src={`${config.ownerId}/widget/AAVE.PrimaryButton`}
+                      props={{
+                        children: "Supply",
+                        onClick: () => {
+                          State.update({ data: row });
+                          setShowSupplyModal(true);
+                        },
+                      }}
+                    />,
+                  ]),
+                }}
+              />
+              {/* mobile view */}
+              {assetsToSupply.map((row) => {
+                return (
+                  <Widget
+                    src={`${config.ownerId}/widget/AAVE.Card.CardContainer`}
+                    props={{
+                      children: [
+                        <Widget
+                          src={`${config.ownerId}/widget/AAVE.Card.CardsBody`}
+                          props={{
+                            config,
+                            children: [
+                              <Widget
+                                src={`${config.ownerId}/widget/AAVE.Card.TokenWrapper`}
+                                props={{
+                                  children: [
+                                    <img
+                                      width={64}
+                                      height={64}
+                                      src={`https://app.aave.com/icons/tokens/${row.symbol.toLowerCase()}.svg`}
```

Comment:
We'd better not rely on AAVE app site for the icons. But I think we can change this in another PR.